content/en/docs/concepts/windows/intro.md

```diff
@@ -352,6 +352,38 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Hardware recommendations and considerations {#windows-hardware-recommendations}
+
+{{< note >}}
```

Comment:
Since there are links to microsoft and a forum post, we may need a thirdparty content label, see https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#third-party-content-marker

```suggestion
## Hardware recommendations and considerations {#windows-hardware-recommendations}
{{% thirdparty-content %}}

{{< note >}}
```